cip.go

```diff
@@ -131,6 +147,16 @@ func main() {
 	}
 	sc.ReadDigestsAndTags(mkRegistryListTagsCmd)
 
+	if *createManifestPtr {
+		if err := sc.CreateManifest(mfest, *createMfestOutputPtr); err != nil {
+			fmt.Fprintln(
+				os.Stderr,
+				fmt.Errorf("cannot create manifest: %s", err))
+		}
+
+		return
```

Comment:
I would prefer to `os.Exit(0)` here to make it more explicit that we are stopping execution entirely.

Reply:
Done

Reply:
Done